### PR TITLE
chore: update IP addresses

### PR DESCRIPTION
**Pull request by omjadas**

### CODE REVIEW

File: whitelist.yml [hunk 1]
```diff
@@ -2,18 +2,31 @@
...
   - 52.23.148.198
   - 52.4.18.170
-nz:
-  - 103.16.181.211
+  - 107.21.125.147
+  - 54.85.120.50
+  - 52.45.11.42
+  - 54.84.197.145
+  - 52.204.99.5
+  - 18.213.233.108
```
Comment (hreisi) [inline whitelist.yml:26]: Where does this IP come from?
18.213.233.108

File: whitelist.yml [hunk 2]
```diff
@@ -2,18 +2,31 @@
...
   - 52.62.3.168
+  - 54.66.51.34
+  - 54.79.207.8
+  - 52.65.195.15
+  - 52.62.240.64
+  - 54.252.43.78
+  - 54.79.124.27
 eu-central:
-  - 52.29.64.156
-  - 18.196.195.93
```
Comment (hreisi) [inline whitelist.yml:7]: Do we want to remove these two ip addresses a bit later so that we can roll back easily?
52.29.64.156
18.196.195.93

Comment (omjadas) [reply whitelist.yml:7]: these are not being used

File: whitelist.yml (hunk 1 above)
Comment (omjadas) [reply whitelist.yml:26]: that is one of the new Staging USA IP addresses

**Review by hreisi — APPROVED** [review]

**Pull request merged**